### ISSUE TITLE
Clips or Hours? ...how we communicate data going forward

### ISSUE THREAD

**Issue by m-branson** [mozilla/voice-web#1167]
Currently we collect and validate `clips`, these translate *magically* to `hours`. The overall goals of the Common Voice project are measured in hours, though we are currently counting and measuring `clips` as user level stats.

**Suggestion: move to time based measurements across the project (including user stats).**

@mikehenrty @Gregoor @kdavis-mozilla what are the implications of doing this? Is it technically feasible?

cc @flekkowich @Gregoor @LRSaunders

**Comment by mikehenrty** [comment]
> @mikehenrty @Gregoor @kdavis-mozilla what are the implications of doing this? Is it technically feasible?

Definitely technically feasible as we just estimate by average length of clip. This is what we currently do for the homepage progress bar for instance.

After giving this some though, I feel using hours/minutes vs clip count is better since it informs the user of the sense of scale we need. It might be a little disillusioning when their profile clip count changes from, say 2000 clips, to 2.5 hours. But I think the switch is worth it to not hide our goals and how we are actually making progress towards them.

Long story short, I agree with your suggestion :)

**Comment by Gregoor** [comment]
I'm with y'all as well on reducing cognitive load on this, though I do wonder if showing user's contribution in time (opposed to clip count) would incentivize users recording longer clips 😁

**Comment by mikehenrty** [comment]
> This flag that you raise @Gregoor is a great one, is there a way we can measure and understand if this starts to happen from an analytics perspective?

@m-branson, two things we can do to mitigate. one, make it clear that longer recordings don't actually give you more time on your stats (since it's only an average). two, we can definitely look if the average clip length starts to go up asymptotically.

**Comment by connorshea** [comment]
For what it's worth, as a user I kind of feel like this might be demotivating because I could have 100 clips, which feels like a lot, but it'd only be 7.5 minutes of recordings. It would also mean that each recording session of 5 clips wouldn't add to my overall total by very much.

I'd prefer showing both, if possible :)

**Comment by kdavis-mozilla** [comment]
Does it have to be one or the other? Could it be both clips and hours?

**Comment by m-branson** [comment]
With regard to my statement here:

>I recommend using a traditional 00:00:00 timestamp down to the second to help convey how each recording is moving a stat or goal needle.

This has been furthered discussed via #1325 and I now revise my recommendation for time based UI to:

>I do feel this [00:00:00] is the most accurate route, though it could be a more belabored route. Perhaps we balance the two approaches and utilize a 00h 00m display method to account for various UI needs across the site.
>
>This longer format would require more horz space to accommodate label & info though I have an idea of how to achieve that layout should we decide to go this route across the entire site (which I heartily advocate).

**Comment by Gregoor** [comment]
Sounds good to me!

> • This format will occur most consistently in the UI as 00h 00m format where 00h does not display if ≤ 59min have been achieved.

@m-branson And similarily 00m does not displayi f >= 100h?

**Comment by m-branson** [comment]
>@m-branson And similarily 00m does not displayi f >= 100h?

@Gregoor yes please :)

@mikehenrty wanted to make sure you saw my question here due to our trello bot sync oddity:

>• In the contribution portal, we will still utilize a 1/5 count to indicate individual recording and validation segments. The submission feedback screen should also still reflect a percentage of total goal where the goal will now read as a time based measurement. @mikehenrty do you have a breakdown for daily goal size?
